container/bridge/src/test/java/org/opennms/container/web/felix/base/internal/handler/ServletHandlerRequestTest.java

```diff
@@ -92,11 +90,12 @@ public ServletHandlerRequestTest(final HttpServletRequest request, final String
     }
 
     @Test
+    // TODO MVR fix me
```

Comment:
TODO

Reply:
Yeah there is still some more work to do, I just wanted to have it as a PR so I can find it more easily (-: